terraform/modules/configs/deploy-all/main.tf

```diff
@@ -159,6 +159,37 @@ data "aws_ssm_parameter" "browser_rollbar_access_token" {
 
 data "aws_ssm_parameter" "lb_public_alb_arn" {
   name = "${lower(var.environment)}-lb-public-alb-arn"
+
```

Comment:
Getting a compile error as closing brace missing

Reply:
Issue should be resolved now